packages/tools/src/apm/install.coffee.md

```diff
@@ -53,3 +53,11 @@ Install Atom packages with APM.
           if: install.length
         , (err) =>
           @log message: "APM Installed Packages: #{install.join ', '}"
+
+## Exports
+
+    module.exports = install
```

Comment:
Please respect the conventions this is not how the others actions are written

Reply:
You mean assign the function directly to `module.exports` rather than assigning to the `install` variable first then `module.exports` later?

If not, could you be more specific?